tests/test_karmabot.py

```diff
@@ -217,6 +221,66 @@ def test_create_karma_user(mock_empty_db_session, mock_slack_api_call):
     assert second.username == "clamytoe"
 
 
+# KarmaNote
+def test_karma_note_add(mock_filled_db_session):
```

Comment:
Great that you added tests, thanks. I think we should break out the commands into a separate test module later on, what do you think @pogross?